commands/plugin_e2e_test.go

```diff
@@ -22,6 +22,82 @@ func (s *MmctlE2ETestSuite) TestPluginAddCmd() {
 
 	pluginPath := filepath.Join(os.Getenv("MM_SERVER_PATH"), "tests", "testplugin.tar.gz")
 
+	s.RunForSystemAdminAndLocal("add an already installed plugin without force", func(c client.Client) {
+		printer.Clean()
+
+		s.th.App.UpdateConfig(func(cfg *model.Config) {
+			*cfg.PluginSettings.Enable = true
+			*cfg.PluginSettings.EnableUploads = true
+		})
+
+		defer s.th.App.UpdateConfig(func(cfg *model.Config) {
+			*cfg.PluginSettings.Enable = false
+			*cfg.PluginSettings.EnableUploads = false
+		})
+
+		err := pluginAddCmdF(c, &cobra.Command{}, []string{pluginPath})
+		s.Require().Nil(err)
+
+		s.Require().Equal(1, len(printer.GetLines()))
+		s.Require().Contains(printer.GetLines()[0], "Added plugin: ")
+
+		err = pluginAddCmdF(c, &cobra.Command{}, []string{pluginPath})
+		s.Require().Nil(err)
+
+		s.Require().Equal(1, len(printer.GetLines()))
```

Comment:
I think we should clean the `printer` after we first install the plugin successfully then check that printer does not have any regular output in case of an error.